bundles/action/org.openhab.action.homematic/META-INF/MANIFEST.MF

```diff
@@ -10,7 +10,10 @@ Bundle-Activator: org.openhab.action.homematic.internal.HomematicActivator
 Bundle-ManifestVersion: 2
 Bundle-Description: This is the Homematic action of the open Home Aut
  omation Bus (openHAB)
-Import-Package: org.openhab.core.items,
+Import-Package: org.openhab.binding.homematic.internal.common,
+ org.openhab.binding.homematic.internal.communicator.client,
```

Comment:
Actually, there shouldn't even a dependency from an action bundle to a binding bundle at all. The shared code should rather be externalized into an io bundle as it is e.g. done for squeezebox.